keyboard/src/lib/y2keyboard/keyboards.rb

```diff
@@ -340,24 +340,23 @@ def self.optional_keyboards
     # memoize this
     return @optional_keyboards unless @optional_keyboards.nil?
 
-    @optional_keyboards = []
+    localectl_keymaps = `localectl list-keymaps`.split("\n")
```

Comment:
Can we rely on `localectl` always being available on all products?

I know it's part of the _systemd_ package, at least on my Leap 15.4. Can we / do we want to rely on that?

Or maybe check here if the command is actually there?